.husky/pre-commit

```diff
@@ -1 +1,20 @@
-npm run lint
\ No newline at end of file
+echo 'Check for linting errors...'
```

Comment:
**SUGESTÃO:**
_(não precisa fazer se não quiser)_

Caso a gente venha a incrementar esse arquivo, seria legal separar ele por funções, e caso precisasse adicionar alguma coisa seria mais simples de dar a manutenção. Algo como:

`
check_linting_errors() {
    echo 'Check for linting errors...'
    npm run check || {
        echo 'Prettier errors failed. Please fix them before commiting.'
        exit 1
    }

    npm run lint || {
        echo 'Linting errors failed. Please fix them before commiting.'
        exit 1
    }

    echo 'Linting completed successfully.'
}

check_modified_files() {
    local file_count=$(git diff --cached --name-only | wc -l)
    local max_files=3

    if [ $file_count -gt $max_files ]; then
        echo "ERRO: Mais de três arquivos modificados no commit."
        exit 1
    fi
}

check_linting_errors
check_modified_files

echo 'Done'
exit 0
`

Reply:
Concordo com a sugestão da @htamagnus, assim já deixa padronizado para quando precisar implementar novas regras.